npm/cypress-schematic/src/schematics/ng-generate/cypress-ct-tests/index.spec.ts

```diff
@@ -30,11 +31,9 @@ describe('@cypress/schematic:e2e ng-generate', () => {
     appTree = await schematicRunner.runExternalSchematicAsync('@schematics/angular', 'application', appOptions, appTree).toPromise()
   })
 
-  it('should create cypress spec file', async () => {
-    return schematicRunner.runSchematicAsync('e2e', { name: 'foo', project: 'sandbox' }, appTree).toPromise().then((tree) => {
-      const files = tree.files
-
-      expect(files).to.contain('/projects/sandbox/cypress/e2e/foo.cy.ts')
-    })
+  it('should create cypress component tests alongside components', () => {
+    schematicRunner.runSchematicAsync('specs-ct', { createSpecs: true, project: 'sandbox' }, appTree).pipe(
```

Comment:
```suggestion
    await schematicRunner.runSchematicAsync('specs-ct', { createSpecs: true, project: 'sandbox' }, appTree).toPromise()
```

I've had to utilize the `done` mocha callback in the past when working with observables to ensure the test executes properly. Using the `toPromise` makes the test lest nested and you don't have to worry about explicitly calling `done`

Same comment for the other schematic unit tests

Reply:
I left one of these tests as-is because it fails using `toPromise` because of how the files are generated by the schematic. I talked through it with @jordanpowell88. All of the other tests were fine.